apps/src/lib/kits/maker/commands.js

```diff
@@ -1,14 +1,41 @@
 /** @file Maker commands (invoked by Applab/Gamelab.executeCmd) */
 import {
   apiValidateType,
-  apiValidateTypeAndRange
+  apiValidateTypeAndRange,
+  outputError
 } from '../../util/javascriptMode';
 import {BOARD_EVENT_ALIASES} from './boards/circuitPlayground/PlaygroundConstants';
 import MicroBitBoard from './boards/microBit/MicroBitBoard';
 
 /** @private {CircuitPlaygroundBoard} */
 let board;
 
+function isValidPin(pin) {
```

Comment:
Hmm so looking at this function again, I'm thinking it may be cleaner to just include this logic in `commands` under `apiValidateType()`? That way the logic is centralized to one place and it's associated with the similar output warning logic. I know we talked about keeping the error logic in this file because we need to early return in the case of an error, but I think we can use the return value `apiValidateType` to similarly early return here? If not though, then I'm also fine with this logic staying in this file.

Reply:
Thanks! Just moved logic to check for invalid pinid to apiValidateType.

Reply:
Also updated tests and used valid pin ids for maker command calls.